resources/lang/es/gitscrum.php

```diff
@@ -8,184 +8,184 @@
 
 
 return[
-    'title-login' => 'GitScrum Login',
-    'welcome-to' => 'Welcome to',
-    'edit-comment' => 'Edit Comment',
-    'configuration' => 'Configuration',
-    'configuration-config-status-' => 'Configuration - Config Status - ',
+    'title-login' => 'Acceder a GitScrum',
+    'welcome-to' => 'Bienvenido a',
+    'edit-comment' => 'Editar comentario',
+    'configuration' => 'Configuración',
+    'configuration-config-status-' => 'Configuración - estatus - ',
```

Comment:
i think _estatus_ and _status_ are pretty interchangeable in Spanish. I tend to understand _estatus_ as the spanish version of the original latin word 'status'. Or at least, that's what the RAE popes preach in the spanish _bible_ ... http://dle.rae.es/
Sí... ¿ es  _estatus_  el término hispanico ? @m1guelpf @ortegacmanuel